src/secure_random.cr

```diff
@@ -92,7 +101,7 @@ module SecureRandom
     @@initialized = true
 
     {% if flag?(:linux) %}
-      if getrandom(Bytes.new(16)) >= 0
+      if sys_getrandom(Bytes.new(16)) >= 0
```

Comment:
Well, i've saved an old behaviour:
before commit was 1:`getrandom(Bytes):Int` that returns number of bytes and 2:`getrandom(Int):Bytes` that wraps (1) for arbitrary size.
Now (2) has signature `getrandom(Bytes):Nil`, so (1) is renamed to `sys_getrandom(Bytes)` to avoid conflict. Here we are checking if getrandom syscall is supported, so `sys_getrandom` seems right function for me.